index.md

```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+>
+> *A Company*
 
-Some more text.
+Get in touch today to discuss getting these seminars given to you. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) or ring me on 07948 284729.
 
 ## The Course
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+The course is designed to cover the key areas of modern software engineering best practises. Each module will help developers to adopt industry standard approaches to developing software. These cover both technical topics such as writing automated tests, as well as procedural topics such as en effective code review process.
+
+You may already be using some of these techniques, so the course is entirely customsisable to exclude the parts you already do, or provide more details on what is mission critical to you.
+
+**Get in touch today to craft a bespoke course for your business.**
+
+The full course consists of 5 modules:
 
  1. [Source Control](#source-control)
  2. [Automated Tests](#automated-tests)
  3. [“Good” Code](#good-code)
  4. [Code Review](#code-review)
- 5. [Continuous Integration (CI)](#continuous-integration-ci) 
+ 5. [Continuous Integration (CI)](#continuous-integration-ci)
 
+ Each of the talks features practical examples using Python, [Git](https://git-scm.com/), Gitlab. After each talk I will answer questions - giving plenty of time for the specific challenges you are having using these best practises. I also like to foster an on-going relationship after the talks to help you put these tools and techniques into practise.
 
 ### Souce Control
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+Source control is a tool that stores your source code, and tracks changes to that code across time. Popular source control software includes [Git](https://git-scm.com/) and [Subversion](https://subversion.apache.org/).
```

Comment:
More focus on why they would want to learn/use source control rather than what it is?